### PR TITLE
DRAFT: EVEREST-839-multinamespace-e2e-tests

### PR DESCRIPTION
**Pull request by solovevayaroslavna**

### CODE REVIEW

File: apps/everest/.e2e/multinamespaces/db-wizard.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    await expect(
+      page.getByTestId('postgresql-toggle-button')
+    ).not.toBeVisible();
+    await expect(page.getByTestId('text-input-db-name')).toHaveValue(
+      /.*mysql.*/
+    );
+
+    let dbVersion = '';
+
+    page
```
Comment (fabio-silva) [inline apps/everest/.e2e/multinamespaces/db-wizard.e2e.ts:53]: shouldn't this just be
```
const dbVersion = await page.getByTestId('select-input-db-version').inputValue();
```
?

File: apps/everest/.e2e/multinamespaces/db-wizard.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    await expect(page.getByTestId('mongodb-toggle-button')).toBeVisible();
+    await expect(
+      page.getByTestId('postgresql-toggle-button')
+    ).not.toBeVisible();
+    await expect(page.getByTestId('mysql-toggle-button')).not.toBeVisible();
+    await expect(page.getByTestId('text-input-db-name')).toHaveValue(
+      /.*mongodb.*/
+    );
+
+    await page
```
Comment (fabio-silva) [inline apps/everest/.e2e/multinamespaces/db-wizard.e2e.ts:76]: same

File: apps/everest/.e2e/multinamespaces/monitoring.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  test.beforeAll(async ({ request }) => {
+    token = await getTokenFromLocalStorage();
+    await createBackupStorageFn(request, pxcStorageLocationName, [
+      EVEREST_CI_NAMESPACES.PXC_ONLY,
+    ]);
+  });
+
+  test.afterAll(async ({ request }) => {
+    await deleteStorageLocationFn(request, pxcStorageLocationName);
+    // await deleteMonitoringInstance(request, psmdbMonitoringEndpoint, token);
```
Comment (fabio-silva) [inline apps/everest/.e2e/multinamespaces/monitoring.e2e.ts:43]: can this be removed?

File: apps/everest/.e2e/utils/backup-storage.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+} = process.env;
+
+export const createBackupStorageFn = async (
+  request: APIRequestContext,
+  name: string,
+  namespacesList?: string[]
+) => {
+  const token = await getTokenFromLocalStorage();
+  const namespaces: string[] = namespacesList
+    ? namespacesList
```
Comment (fabio-silva) [inline apps/everest/.e2e/utils/backup-storage.ts:19]:
```suggestion
    ??
```

File: apps/everest/.e2e/utils/table.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { Page } from '@playwright/test';
+
+export const findRowAndClickActions = async (
+  page: Page,
+  name: string,
+  nameOfAction?: string,
+  before?: () => {}
```
Comment (fabio-silva) [inline apps/everest/.e2e/utils/table.ts:7]: what is `before`?

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: apps/everest/.e2e/multinamespaces/db-wizard.e2e.ts (hunk 2 above)
Comment (solovevayaroslavna) [reply apps/everest/.e2e/multinamespaces/db-wizard.e2e.ts:76]: fixed

File: apps/everest/.e2e/utils/table.ts (hunk 5 above)
Comment (solovevayaroslavna) [reply apps/everest/.e2e/utils/table.ts:7]: thanks! i forgot to remove it after changes

**Review by fabio-silva — APPROVED** [review]

**Pull request merged**